src/pages/en/guides/markdown-content.md

```diff
@@ -178,113 +78,107 @@ For Markdown files, the `content` prop also has an `astro` property which holds
 }
 ```
 
-> Keep in mind that the only guaranteed properties coming from the `content` prop are `astro` and `url`.
+> 💡 `astro` and `url` are the only guaranteed properties provided by Astro in the `content` prop. The rest of the object is defined by your front matter variables.
 
-### Images and videos
+### Variables 
+
+Front matter variables can be used directly in your Markdown as properties of the `frontmatter` object.
+
+```markdown
+---
+layout: ../layouts/BaseLayout.astro
+author: Leon
+age: 42
+---
+# About the Author
 
-Using images or videos follows Astro’s normal import rules:
+{frontmatter.author} is {frontmatter.age} and lives in Toronto, Canada.
```

Comment:
Im pretty certain that this is like:
```suggestion
{author} is {age} and lives in Toronto, Canada.
```

Reply:
and if we are going with fictional people, could we bring back Andy Kaufman, and have him living on the moon?

Reply:
I know I snagged this example from somewhere, and this was how it was written. Can't remember if I tested it for myself yet, so I'll do that!

Reply:
I know the blog starter uses `{frontmatter.prop}` and that works.

Reply:
That must be where I grabbed it from, and I just double checked, and that's in fact what they do.

(I also kind of hope it works that way, because it's a nice symmetry with the `content.` object stuff.)